features/multipication.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Multiplication
+  As a mathematician
+  I want my calculator to multiply numbers
+  So I don't have to
+
+   Scenario: Multiply numbers to get a product
+    When I multiply two numbers
+    Then I get the product
+
+   Scenario: Multiply by zero
+  	When I multiply a number by zero

```

Comment:
Spacing is a little off here. Ruby is a 2 space (no tabs) language by convention. Because you've got it mostly right in your files, I suspect that this might have been an editor issue. Just be on the lookout for it. No points were taken off. Just some feedback.